routing/unified_edges_test.go

```diff
@@ -114,6 +123,21 @@ func TestNodeEdgeUnifier(t *testing.T) {
 			unifier:          unifierNoInfo,
 			expectedCapacity: 0,
 		},
+		{
+			name:           "local insufficient bandwidth",
+			unifier:        unifierLocal,
+			amount:         200,
+			expectNoPolicy: true,
+		},
+		{
+			name:             "local",
+			unifier:          unifierLocal,
+			amount:           100,
+			expectedFeeBase:  p1.FeeBaseMSat,
+			expectedFeeRate:  p1.FeeProportionalMillionths,
+			expectedTimeLock: p1.TimeLockDelta,
+			expectedCapacity: 0,
```

Comment:
@bitromortac here I am adding coverage for the change introduced in 6857. Is zero indeed expected here?